src/server.ts

```diff
@@ -21,7 +22,16 @@ interface ServerConfig {
   url: string
 }
 
-export function startServer(config: ServerConfig): Promise<Server> {
+interface Context {
+  entityService: EntityService
+  schemaService: SchemaService
+  submissionService: SubmissionService
+}
+
+export function startServer(
+  config: ServerConfig,
+  context: Context,
```

Comment:
I'm not a big fan of the word `context`, but I can't think of something else. I would say `services`, but then we would be calling `services.submissionService` and that doesn't sound great either.

Reply:
But you're right it's not a good name if it's defnied a used in the server module.